### ISSUE TITLE
Next steps (December 2024 and forward)

### ISSUE THREAD

**Issue by GBKS** [GBKS/opendesign.guide#27]
In our reading club session [last week](https://github.com/BitcoinDesign/Meta/issues/734), we read through the 12th and [final chapter](https://opendesign.guide/12-final-project) in the guide. The goal of this club was to do a first revision of the AI-generated content, which we have now done. Big shout-out and thanks to everyone who joined in, whether you participated in any calls, commented somewhere or otherwise.

**Implementing feedback**
Throughout the time we had the sessions, I've been taking time to take our collective notes and tried to implement them in pull requests to the site. Improvements for chapters 1-4 are already live. Changes to [chapters 5-7](https://github.com/GBKS/opendesign.guide/pulls) need review and approval before they can go live. For chapters 8-12, I still need to create pull requests. Getting these done and live in the next 1-2 month, maybe by the end of January, will be a really good step forward.

**IRL examples**
In parallel, I think we should tackle another issue that has come up many, many times in our calls. Everyone wants to see more resources, examples, and case studies. Without them, it's too theoretical. We want stories about what fascinates designers in open-source, how they go about their work, real-life stories from projects they have completed, how projects organize design work, etc. Chapter 11 is called case studies, but this goes beyond just that chapter. [Case studies](https://bitcoin.design/guide/case-studies/) are more long-form and involved examples, where we have lot of opportunities for quotes and external references in the other chapters.

I think there are several ways to start this one:
- Look at your own work history and find examples that could fit
- Ask people around you, or communities you are part of
- Review the content to see which parts would benefit from examples
- AI generate fake examples (just kidding, don't do this)

We can use issue #3 to track this work.

**Promoting the project**
[FOSS Backstage](http://foss-backstage.de) and [FOSDEM](http://fosdem.org) are coming up early in 2025. With dedicated design tracks, and their focus on open-source, these are great opportunities to talk about this project. Maybe it will help new designs become more comfortable more quickly, or even help experienced designers fill in some gaps. Can you think of other conferences, meetups or events? Everyone is welcome to help out with this one. The world is your oyster.

**Use the guide**
Share it with designers, teach it in a bootcamp, online class, or otherwise. That's what it's meant for.

**Illustrations & other**
Really, you can contribute in any way you want. If you browse the guide and see a way to improve it, bring it up and make it happen.

So that's what I'd recommend we do. What do you think? I'm totally cool if we do something different as well.

**Comment by Erioldoesdesign** [comment]
mmm I think what I would love to do is trial this with design students. I have some cold-contacts and connections in design schools and then maybe some warmer contacts from previous workshops I've given. I think maybe what I'll start with is some email copy/social media copy to promo and then see what it looks like to push it forward.

I might also see how I can develop it locally in my country too, through meet-ups etc.

**Comment by GBKS** [comment]
Got word from [FOSS Backstage](http://foss-backstage.de) that my proposal for a talk about this project got approved. Woohoo!

Anyone else going and wants to tag team? It is in-person, in Berlin on March 10-11.

**Comment by Erioldoesdesign** [comment]
> Got word from [FOSS Backstage](http://foss-backstage.de) that my proposal for a talk about this project got approved. Woohoo!
> 
> Anyone else going and wants to tag team? It is in-person, in Berlin on March 10-11.

I'll be there and very happy to back you up if you'd like!

**Comment by GBKS** [comment]
@Erioldoesdesign thank you, that would be fantastic.
